components/cluster.py

```diff
@@ -23,15 +23,43 @@ def get_data(session_id):
     return eq_data
 
 
-def compute_edges(df):
-    """Return the list of edges to be used to build the clustering graph."""
+def compute_edges(data, df):
```

Comment:
It's kind of weird that the df is given as an argument when the only thing it's used for is to check whether it's empty. Is it really necessary?